ui/src/components/PurseAmountInput.tsx

```diff
@@ -1,12 +1,13 @@
 import { AmountInput, type PurseJSONState } from '@agoric/react-components';
-import type { Amount, AssetKind } from '@agoric/web-components';
+import type { AssetKind } from '@agoric/web-components';
 import { useState } from 'react';
 import { CopyBagEntry, PurseValue, SetEntry } from './DisplayAmount';
 import { stringifyData } from '../utils/stringify';
-import { makeCopyBag } from '@endo/patterns';
+import { makeCopyBag, Key } from '@endo/patterns';
 
 type Props = {
   purse: PurseJSONState<AssetKind>;
+  onChangeInput: (amount: {brand: globalThis.Brand, value: unknown} | null) => void;
```

Comment:
make this optional so you don't have to give it a no-op function when there's nothing to do

Reply:
I see. Good catch.